app/code/Magento/GoogleGtag/Block/Ga.php

```diff
@@ -147,20 +147,22 @@ public function getOrdersTrackingData(): array
         foreach ($collection->getItems() as $order) {
             foreach ($order->getAllVisibleItems() as $item) {
                 $result['products'][] = [
-                    'item_id' => $this->escapeJsQuote($item->getSku()),
-                    'item_name' =>  $this->escapeJsQuote($item->getName()),
-                    'price' => number_format((float) $item->getPrice(), 2),
-                    'quantity' => (int)$item->getQtyOrdered(),
+                    'item_id' => $this->_escaper->escapeHtml($item->getSku()),
+                    'item_name' =>  $this->_escaper->escapeHtmlAttr($item->getName()),
```

Comment:
Please update `$this->_escaper->escapeHtml` instead of `$this->_escaper->escapeHtmlAttr` here also.